plone/app/layout/viewlets/common.py

```diff
@@ -201,7 +201,8 @@ def update(self):
         self.selected_tabs = self.selectedTabs(portal_tabs=self.portal_tabs)
         self.selected_portal_tab = self.selected_tabs['portal']
 
-    def selectedTabs(self, default_tab='index_html', portal_tabs=()):
+    def selectedTabs(self, default_tab='index_html', portal_tabs=None):
+        portal_tabs = portal_tabs or []
```

Comment:
I am guessing the original tuple was fine too, because it is immutable, but this sure looks safer.